ledger/eval.go

```diff
@@ -572,38 +574,17 @@ func (eval *BlockEvaluator) transaction(txn transactions.SignedTxn, ad transacti
 			return fmt.Errorf("transaction %v: malformed: %v", txn.ID(), err)
 		}
 
-		// Properly signed?
-		if eval.txcache == nil || !eval.txcache.Verified(txn) {
-			err = verify.TxnPool(&txn, spec, eval.proto, eval.verificationPool)
+		if eval.txcache == nil || !eval.txcache.Verified(txn, ctx.Params) {
```

Comment:
We need to be sure that `ctx.CurrProto` is equal to `eval.proto` above -- it might be nice if we constructed `ctx.Params` right here for clarity

Reply:
Actually, if we're ditching `FirstValidTime`, we can eliminate `PrepareContexts` entirely since the `Params` should be the same for every txn